Dockerfile

```diff
@@ -53,11 +53,15 @@ RUN mkdir -p /tmp/run_artifacts
 # download benchmark-operator to /tmp default path
 RUN git clone https://github.com/cloud-bulldozer/benchmark-operator /tmp/benchmark-operator
 
+# download cluster-buster to /tmp default path && install cluster-buster dependency
+RUN git clone https://github.com/RobertKrawitz/OpenShift4-tools /tmp/OpenShift4-tools \
```

Comment:
I want to decide on how to specify the correct version of ClusterBuster to use (simply the tip of main is not enough).